terraform/build-windows.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+instance_id=$1
+if [[ $2 == main ]]; then
+    version=main
+else
+    version=v$2
+fi
```

Comment:
a bit of a weird assignment, you could do:

```suggestion
version=$2
if [[ $version != main ]]; then
    version=v$version
fi
```